README.md

```diff
@@ -1,336 +1,79 @@
 # Lightning Loop
- 
 Lightning Loop is a non-custodial service offered by
-[Lightning Labs](https://lightning.engineering/) to bridge on-chain and
-off-chain Bitcoin using submarine swaps. This repository is home to the Loop
-client and depends on the Lightning Network daemon
-[lnd](https://github.com/lightningnetwork/lnd). All of lnd’s supported chain
-backends are fully supported when using the Loop client: Neutrino, Bitcoin
-Core, and btcd.
+[Lightning Labs](https://lightning.engineering/) that makes it easy to move
+bitcoin into and out of the Lightning Network.
 
-In the current iteration of the Loop software, two swap types are supported:
-  * off-chain to on-chain, where the Loop client sends funds off-chain in
-  * on-chain to off-chain, where the Loop client sends funds to an on-chain
-    address using an off-chain channel
+## Features
+- Automated channel balancing
+- Privacy-forward non-custodial swaps
+- Opportunistic transaction batching to save on fees
+- Progress monitoring of in-flight swaps
 
-We call off-chain to on-chain swaps, a **Loop Out**.  The service can be used
-in various situations:
+## Use Cases
+- Automate channel balancing with AutoLoop ([Learn more](https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md))
+- Deposit to a Bitcoin address without closing channels
+- Convert outbound liquidity into inbound liquidity
```

Comment:
Should we perhaps introduce the swap types here? Otherwise as a first-timer, you might not be sure which one you need.
- Deposit to a Bitcoin address without closing channels with loop in
- Convert outbound liquidity into inbound liquidity with loop out

Reply:
Great idea! Added